src/js/utils/obj.js

```diff
@@ -30,6 +30,17 @@
  */
 const toString = Object.prototype.toString;
 
+/*
+ * Gives keys of an object
+ *
+ * @param {Object} object
```

Comment:
The doc comments should probably look something like this to follow a similar format for the project. I more descriptive language as well.
```js
/**
 * Get the keys of an Object
 *  
 * @param {Object}
 *        The Object to get the keys from
 *
 * @return {string[]}
 *         An array of the keys from the object. Returns an empty array if the 
 *         object passed in was invalid or had no keys.
 * 
 * @private
 */
```